starcluster/commands/start.py

```diff
@@ -264,8 +264,8 @@ def execute(self, args):
                     "Cannot use --dns-prefix when the cluster tag contains "
                     "a dot.")
             scluster.dns_prefix = tag
-        if self.opts.dns_sufix:
-            scluster.dns_sufix = tag
+        if self.opts.dns_suffix:
+            scluster.dns_suffix = self.opts.dns_suffix
```

Comment:
Thinking about it. I'm ok with changing the behaviour, but you need to change the option "help" description to reflect it.

Thanks!